shader/coarse.wgsl

```diff
@@ -401,6 +410,9 @@ fn main(
     }
     if bin_tile_x + tile_x < config.width_in_tiles && bin_tile_y + tile_y < config.height_in_tiles {
         ptcl[cmd_offset] = CMD_END;
-        // TODO: blend stack allocation
+        if max_blend_depth > BLEND_STACK_SPLIT {
+            let scratch_size = max_blend_depth * TILE_WIDTH * TILE_HEIGHT * 4u;
+            ptcl[blend_offset] = atomicAdd(&bump.blend, scratch_size);
```

Comment:
It seems like this needs to set the failure flag?

Reply:
Should it? This can’t really fail as we don’t actually use the allocation until fine. We’re just doing the accounting. But I could be missing something.